org.eclipse.scout.rt.dataobject/src/main/java/org/eclipse/scout/rt/dataobject/id/IdCodec.java

```diff
@@ -40,6 +43,14 @@ public class IdCodec {
   protected final Map<Class<?>, Function<String, Object>> m_rawTypeFromStringMapper = new HashMap<>();
   protected final Map<Class<?>, Function<Object, String>> m_rawTypeToStringMapper = new HashMap<>();
 
+  public interface IIdCodecFlag {
+  }
+
+  public enum IdCodecFlag implements IIdCodecFlag {
+    LENIENT,
```

Comment:
Javadoc indicating behavior of flag, for LENIENT see former javadoc of ...lenient() methods